aws/data_source_aws_rds_cluster.go

```diff
@@ -263,9 +265,14 @@ func dataSourceAwsRdsClusterRead(d *schema.ResourceData, meta interface{}) error
 		return fmt.Errorf("error setting vpc_security_group_ids: %s", err)
 	}
 
-	// Fetch and save tags
-	if err := saveTagsRDS(conn, d, aws.StringValue(dbc.DBClusterArn)); err != nil {
-		log.Printf("[WARN] Failed to save tags for RDS Cluster (%s): %s", aws.StringValue(dbc.DBClusterIdentifier), err)
+	tags, err := keyvaluetags.RdsListTags(conn, *arn)
```

Comment:
Nit: To prevent potential panics, we generally recommend using the AWS Go SDK provided conversion functions to dereference pointers, e.g. `aws.StringValue(dbc.DBClusterArn)` above